src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -89,6 +89,12 @@ provideEditingConfig(Homepage, {
   properties: [...defaultPageProperties],
   initialContent: {
     ...defaultPageInitialContent,
+    siteColorPrimary: '#274486',
+    siteColorPrimaryDarken: '#1E3466',
+    siteColorPrimaryLighten: '#3054A6',
+    siteColorSecondary: '#39a9eb',
+    siteColorSecondaryDarken: '#1798E4',
+    siteColorSecondaryLighten: '#5EB9EF',
```

Comment:
Why do we need this? The value `274486` is already spread between `index.scss` and `SiteColorsPicker.tsx`. I would prefer not to add one more place.

Reply:
Working on #587 I realized that if the designer changes the default primary and secondary color *and* the color content is unset, the SiteColorsPicker shows the wrong color.
1. This change prevents the unset state for new homepages.
2. The todo in the description prevents the unset state for initial content.

We can probably stick with (2) and revert (1) since homepages are typically created as copies.

Reply:
One understanding question:

> if the designer changes the default primary and secondary color

By that you mean, the designer adjusted `index.scss` and forgot to adjust `SiteColorsPicker.ts` as well?

Reply:
I don't think that adding even more places, that a designer would have to adjust is a good idea. So given we have the color in the css, a default color in javascript and a default color in the content - if the designer only changes the css variable it would still only show the default color because of the content.

Because of that I would simply revert [35684d5](https://github.com/Scrivito/scrivito-portal-app/pull/588/commits/35684d5a296f8ee37fc70b9093c1abbefcd4ab92).